storage.go

```diff
@@ -569,10 +569,15 @@ func (s *Storage) writeAppend(ctx context.Context, o *Object, r io.Reader, size
 }
 
 func (s *Storage) writeMultipart(ctx context.Context, o *Object, r io.Reader, size int64, index int, opt pairStorageWriteMultipart) (n int64, part *Part, err error) {
-	if o.Mode&ModePart == 0 {
+	if !o.Mode.IsPart() {
```

Comment:
We has implement https://github.com/aos-dev/go-storage/pull/573, so we don't need to check them manually.

Reply:
We can return all mode checks like this one.